app/models/article.rb

```diff
@@ -5,4 +5,5 @@ class Article < ApplicationRecord
 
   validates :title, presence: true
   validates :body, presence: true, length: { minimum: 10 }
+  scope :sorted_by, ->(field, order) { order(field => order) }
```

Comment:
лучше вместо order наверное другое слово использовать в переменной
```suggestion
  scope :sorted_by, ->(field, sort_type) { order(field => sort_type) }
```